### ISSUE TITLE
How to use evaluation indicators (ap, ar) instead of auc, pck

### ISSUE THREAD

**Issue by mywordwin** [open-mmlab/mmpose#2344]
Thanks for your work and support.
I want to ask how to evaluate my model on another evaluation metric like mAP, mAR.
For example, the current evaluation indicators in litehrnet_coco_wholebody_hand are pck and AUC, but I want to obtain mAP and mAR indicators, how to modify them?

![image](https://github.com/open-mmlab/mmpose/assets/39481420/a96566ee-53d3-4f8a-96b6-82e76f07a682)

**Comment by Ben-Louis** [comment]
Hi, thanks for using MMPose. We suggest using the new version of MMPose (v1.0), in which you can easily specify evaluation metrics in configs such as https://github.com/open-mmlab/mmpose/blob/dab2fdf09dc2ef21f4725d7549469a28cc718da2/configs/body_2d_keypoint/topdown_heatmap/coco/td-hm_hrnet-w32_8xb64-210e_coco-256x192.py#L146-L150

In MMPose v0.29, you may need to manually replicate the evaluation code from dataset classes employing the mAP metric into your chosen dataset class.

**Comment by mywordwin** [comment]
@Ben-Louis 
Hello, I have used mmpose1.0 at present, and I have replaced val_evaluator. I want to change the evaluation index auc to ap, etc. The hand has 21 key points, and cocoeval defaults to the body with 17 key points. May I ask? Need to modify the cocoeval.py function to make the key points correspond? Or how to do it?
Thank you for your reply, I wish you a happy life!
![image](https://github.com/open-mmlab/mmpose/assets/39481420/3e8595c2-fcdc-4a14-8f61-1307c8328ef8)
